pkg/deploy/generator/scripts/util-services.sh

```diff
@@ -84,26 +92,32 @@ WantedBy=multi-user.target
 # 1) image - nameref, string; RP container image
 # 2) role - nameref, string; VMSS role
 # 3) conf_file - nameref, string; aro rp environment file
+# 4) ipaddress - nameref, string; static ip of podman network to be attached
 configure_service_aro_rp() {
     local -n image="$1"
     local -n role="$2"
     local -n conf_file="$3"
+    local -n ipaddress="$4"
     log "starting"
     log "Configuring aro-rp service"
 
     local -r aro_rp_conf_filename='/etc/sysconfig/aro-rp'
+    local -r add_conf_file="PODMAN_NETWORK='podman'
+IPADDRESS='$ipaddress'
+ROLE='${role,,}'"
 
     write_file aro_rp_conf_filename conf_file true
+    write_file aro_rp_conf_filename add_conf_file false
 
     # shellcheck disable=SC2034
     local -r aro_rp_service_filename='/etc/systemd/system/aro-rp.service'
     # shellcheck disable=SC2034
-    local -r aro_rp_service_file="[Unit]
+    local -r aro_rp_service_file='[Unit]
 After=network-online.target
 Wants=network-online.target
 
 [Service]
-EnvironmentFile=${aro_rp_conf_filename}
+EnvironmentFile=/etc/sysconfig/aro-rp
```

Comment:
I understand why you made this change, as the file is wrapped in single quotes now to prevent expansion of those in the `EnvironmentFile`. I don't really like hard coding the value either though... it's already set in the constant which should be repeated. I don't have a good idea for fixing this at the moment though.

Reply:
The only way I see is to have more variables in the EnvironmentFile having values for the path of these `sysconfig` files for each service and address them in this service file. However, current approach seems to be less complicated than this. WDYT ?

Reply:
I think this is fine, it's not worth the effort and complication to use the same file path variable.